app/views/service_calendars/master_calendar/pppv/totals/_max_total_direct_per_patient.html.haml

```diff
@@ -0,0 +1,34 @@
+-# Copyright © 2011-2016 MUSC Foundation for Research Development
```

Comment:
Extracted to partial from `_total_cost_per_study.html.haml` to rerender. Added column totals and cleaned up